.toys/.toys.rb

```diff
@@ -34,6 +34,13 @@
   t.files = "integration/**/*_test.rb"
 end
 
+expand :minitest do |t|
```

Comment:
Since this is being called internally from `toys samples` (in `.toys/samples.rb` below), consider moving it into that file for encapsulation. (Note: the tool name will change to `toys samples samples-test` when defined in the `samples.rb` file, so you'll need to change the invocation to `exec_tool ["samples", "samples-test"]`.)